packages/create-svelte/templates/default/package.json

```diff
@@ -5,8 +5,8 @@
 	"scripts": {
 		"dev": "svelte-kit dev",
 		"build": "svelte-kit build --verbose",
-		"preview": "svelte-kit preview",
-		"start": "svelte-kit start"
+		"package": "svelte-kit package",
+		"preview": "svelte-kit preview"
```

Comment:
Wow, `start` has been a relic for a while.

I'm curious about the inclusion of `package` though. Is this part of the template that users get with `npm init svelte@next` or is it one of the test apps that get run in CI?

Reply:
this was copy-pasta more than anything — this package.json isn't what ends up in a newly created project, it's just local stuff. package.template.json is where the action is